src/commands/synthetics/__tests__/reporters/default.test.ts

```diff
@@ -135,24 +130,13 @@ describe('Default reporter', () => {
         },
       },
       {
-        description: '1 API test (blocking), 1 location, 3 results: success, failed non-blocking, failed',
+        description: '1 API test, 1 location, 3 results: success, failed non-blocking, failed blocking',
```

Comment:
I'm surprised you could remove the "API test (non-blocking)" case. What happens if the user sets "CI/CD EXECUTION" to "non-blocking" in the /synthetics/details/xxx-yyy-zzz interface? 🤔 Unless this information is now available as the execution rule of the batch? @sdeprez